pkg/gits/bitbucket_cloud.go

```diff
@@ -366,8 +367,8 @@ func (b *BitbucketCloudProvider) UpdatePullRequestStatus(pr *GitPullRequest) err
 	prID := int32(*pr.Number)
 	bitbucketPR, _, err := b.Client.PullrequestsApi.RepositoriesUsernameRepoSlugPullrequestsPullRequestIdGet(
 		b.Context,
-		b.Username,
-		strings.TrimPrefix(pr.Repo, pr.Owner+"/"),
```

Comment:
The logic behind this and other instances of the same is that there were cases where a the `repo` argument was of the form `owner/repo` and it was necessary to trim the `owner/` portion. In any case, `strings.TrimPrefix` is idempotent and so this shouldn't need to be change (and doing so may produce regressions).

Reply:
I can re-review it, but what I found was that the reason it was coming in as `owner/repo` was due to the original PR creation logic that used `FullName` as the `pr.Repo` value. The updated code just uses the `Name` value to set the `pr.Repo`.

Everywhere I could find that this method was used it was getting the value from the `pr.Repo`.

Reply:
Ah okay, that makes sense.